### PR TITLE
Internationalization: Translate TimePickerTitle component

### LINKED ISSUE #53194: Translate TimePickerTitle
As the https://github.com/grafana/grafana/issues/52648 issue is more complex than we initially thought, we divide it into smaller tasks. This task is focused on translating the TimePickerTitle component.

### PR DESCRIPTION
**Pull request by eledobleefe**
**What this PR does / why we need it:**
As the https://github.com/grafana/grafana/issues/52648 issue is more complex than we initially thought, we divide it into smaller tasks. This task is focused on translating the TimePickerTitle component.

**Which issue(s) this PR fixes:**

Fixes https://github.com/grafana/grafana/issues/53194

### CODE REVIEW

File: packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx [hunk 1]
```diff
@@ -122,7 +123,9 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps) {
                   <TimePickerTitle>
                     <Tooltip content={<TooltipContent />} placement="bottom" theme="info">
                       <div>
-                        Specify time range <Icon name="info-circle" />
+                        <Trans i18nKey="time-picker.specify-time-range.title">
```
Comment (joshhunt) [inline packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:126]: can we change this key to `time-picker.time-range.specify` please?

The loose structure we try and keep to is `area.component.message`

**Review by joshhunt — COMMENTED** [review]
just the one key that needs changing, and then this is good to go! nice stuff 😌

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarHeader.tsx [hunk 2]
```diff
@@ -15,7 +16,9 @@ export function Header({ onClose }: TimePickerCalendarProps) {
 
   return (
     <div className={styles.container}>
-      <TimePickerTitle>Select a time range</TimePickerTitle>
+      <TimePickerTitle>
+        <Trans i18nKey="time-picker.calendar-select-range.title">Select a time range</Trans>
```
Comment (eledobleefe) [inline packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarHeader.tsx:20]: @joshhunt according to the structure `area.component.message`, should this key be `time-picker.calendar-header.select-time`? 🤔

File: packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx (hunk 1 above)
Comment (eledobleefe) [reply packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx:126]: yes, of course! 🙂

File: packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarHeader.tsx (hunk 2 above)
Comment (joshhunt) [reply packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarHeader.tsx:20]: hmmm, seeing how basic this component is, i think it's more like `time-picker.calendar.title`?

File: contribute/internationalization.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ const SearchTitle = ({ term }) => (
 
 Prefer using `<Trans />` for JSX children, and `t()` for props and other javascript usage.
 
-When translating in grafana-ui, import `<Trans />` and `t()` from `src/utils/i18n`.
+When translating in grafana-ui, import `<Trans />` and `t()` from `src/utils/i18n` using a relative path.
```
Comment (chri2547) [inline contribute/internationalization.md:33]:
```suggestion
When translating in grafana-ui, use a relative path to import `<Trans />` and `t()` from `src/utils/i18n`.
```

**Review by chri2547 — APPROVED** [review]
One minor suggestion. Thanks!

**Review by joshhunt — APPROVED** [review]

**Pull request merged**